### PR TITLE
July09

### PR DESCRIPTION
**Pull request by tfauck**
Typo in flail for sed command
when cpu 0 is not offline, online fails

### CODE REVIEW

File: fs/flail.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def setUp(self):
         archive.extract(self.get_data("flail-0.2.0.tar.gz"), self.workdir)
         self.build_dir = os.path.join(self.workdir, 'flail-0.2.0')
         os.chdir(self.build_dir)
-        process.system("sed -i 's/.*-lm.*/& -lm' Makefile")
+        process.system("sed -i 's/.*-lm.*/& -lm/' Makefile")
```
Comment (harish-24) [inline fs/flail.py:44]: https://github.com/avocado-framework-tests/avocado-misc-tests/pull/1645

Comment (tfauck) [reply fs/flail.py:44]: sorry for that - I thought it was applied and something got rid of it ....

Comment (harish-24) [reply fs/flail.py:44]: Could you remove this commit?

Comment (tfauck) [reply fs/flail.py]: done

**Pull request merged**